middleware/sse/responder.go

```diff
@@ -37,14 +38,33 @@ func NewResponder(ctx context.Context, eventChan <-chan emitter.Event) Responder
 func (responder Responder) VisitGetJobsEventsResponse(w http.ResponseWriter) error {
 	log := logging.LoggerFromCtx(responder.ctx)
 
-	flusher, _ := w.(http.Flusher)
+	// Check if the ResponseWriter supports hijacking
+	hj, ok := w.(http.Hijacker)
+	if !ok {
+		return fmt.Errorf("hijacking not supported")
+	}
+
+	// use raw connection to prevent it being closed by http.Server's idle cleanup routines
+	conn, bufrw, err := hj.Hijack()
+	if err != nil {
+		return fmt.Errorf("failed to hijack connection: %w", err)
+	}
+	defer func() {
+		_ = conn.Close()
+	}()
 
-	w.Header().Set("Content-Type", "text/event-stream")
-	w.Header().Set("Cache-Control", "no-cache")
-	w.Header().Set("Connection", "keep-alive")
-	w.Header().Set("Access-Control-Allow-Origin", "*")
+	_, _ = fmt.Fprintf(bufrw, "HTTP/1.1 %d\r\n", http.StatusOK)
```

Comment:
Do we really need to limit the event stream to `HTTP/1.1`?

Reply:
Seems so, yes. I should add this to the documentation, noting this as a limitation. In the HTTP/2 case, the interface cast `	hj, ok := w.(http.Hijacker)` would fail (`ok == false`).